Answers.md

```diff
@@ -0,0 +1,19 @@
+1.	How would you teach the basic ideas behind preprocessing to a friend? Please don't copy and paste an answer here, use your own thoughts.
+To a friend, I would explain that preprocessors take information from one programming language and process it into a new input language for another step in the program. There are many different kinds of preprocessors, they are optimized to function for specific languages with purposes like customization, specialization, simplification, or extending a language.  Translating to different language types is to incorporate new functionality while optimizing efficiency of writing syntax components.  We have been using LESS and Node to preprocess more complex design syntax components in our webpages to produce more readable CSS. This helps the CSS also run more efficiently. 
+
+2.	What is the npm command to globally install LESS? 
+Npm install less
```

Comment:
If you want to install something globally, you would have to add a `-g`. So the command would be `npm install -g less`